matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/room/state/StateService.kt

```diff
@@ -103,6 +108,16 @@ interface StateService {
      */
     fun getStateEventsLive(eventTypes: Set<String>, stateKey: QueryStateEventValue): LiveData<List<Event>>
 
+    /**
+     * Get state event containing the IDs of pinned events of the room
+     */
+    fun getPinnedEventsState(): Event?
```

Comment:
I would not add this method in this Service, since this is a shortcut for `getStateEvent`.